W-17167278 : Added note to listener doc (#2732)
mulesoft/docs-connectors: amazon-kinesis/1.0/modules/ROOT/pages/amazon-kinesis-connector-reference.adoc

Uses the Amazon Kinesis Client Library (KCL) to listen from a specified point in a stream.

> NOTE: Due to varying batch sizes, checkpoints might sometimes not occur in the correct sequence, potentially leading to transaction loss in certain cases. Set `maxConcurrency=1` to ensure transactions are processed sequentially.

Reviewer: ```suggestion
NOTE: Due to varying batch sizes, checkpoints can occur in the incorrect sequence, potentially leading to transaction loss in certain cases. Set `maxConcurrency=1` to ensure transactions are processed sequentially.
```

Reviewer 2: Checkpoints might not occur, should I remove "sometimes"?

Reviewer: I tried to avoid the use of "might" and "sometimes" isn’t specific. I changed it a little. Does it work now?
